.appveyor.yml

```diff
@@ -0,0 +1,20 @@
+environment:
+  matrix:
+  - nodejs_version: 8
```

Comment:
Why just 8, and not 4/6/8/9?

Reply:
It runs 2 builds per branch commit, sequentially, and is slower than Circle - we cann add 4/6/9 but it will take hours to finish.

Reply:
Wow lol.  Ok, just 8 it is :)